user-manuals/carrier-management/shipping-box.md

```diff
@@ -0,0 +1,73 @@
+# Shipping Boxes
+
+Retailers employ specific packing boxes tailored to their unique product packaging needs. HotWax Commerce provides users with the ability to define their customized packing boxes. This feature empowers store associates during order fulfillment, allowing them to pack products according to these predefined boxes. These packing boxes are readily accessible for store associates to specify during fulfillment, ensuring accurate specifications that directly impact shipping cost calculations and label generation by the shipping carriers.
+
+This guide outlines a step-by-step process for retailers to establish and manage boxes within HotWax Commerce. Creating customized box configurations ensures precise shipping cost calculations and accurate label generation, aligning with the retailer's specific packaging standards.
+
+Retailers can create and manage the boxes through following ways:
+
+## 1. Creating Boxes in HotWax Commerce
+
+Configuring boxes in the general settings ensures packaging consistency and accuracy throughout the application, aiding store associates in selecting appropriate box during order fulfillment. Incorrect box selection may result in extra shipping charges for retailers. Users can create shipping boxes through following steps:
+
+- Navigate to `HotWax Commerce` > `Hamburger Menu` > `Settings` > `General Settings` Page.
+- Scroll down to find the 'Shipment Box' section.
+- Click 'Add' to open the box creation form and add details such as:
+
+**Shipment Box ID:** Shipment BOX ID with which the boxes will be recognized in HotWax Commerce
+**Description:** Small description of the box, e.g., "Standard Box Square"
+**Dimension Unit of Measurement:** Specify the measurement unit for box dimensions e.g. Inches, Centimetres, etc.
+**Box Dimensions:** Input width, height, and length in the chosen unit. Example: If using inches, entering "10" for box height implies the height of the box is 10 inches.
+
+- Click 'Add' to save the box, the box will now appear in the shipment box section.
+- Use the pencil icon to edit box dimensions if needed.
+
+The added box sizes can now be mapped from in the shipment carrier setup page.
```

Comment:
We can be descriptive here, as why is it important to build the connection with the carrier